settings.gradle.kts

```diff
@@ -4,28 +4,30 @@
 
 @file:Suppress("UnstableApiUsage")
 
+
 rootProject.name = "dokka"
 
 pluginManagement {
     includeBuild("build-logic")
+    includeBuild("build-settings-logic")
 
     repositories {
-        gradlePluginPortal()
         mavenCentral()
+        gradlePluginPortal()
     }
 }
 
 dependencyResolutionManagement {
     repositories {
-        mavenCentral()
-        google()
-
         maven("https://cache-redirector.jetbrains.com/maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-ide")
         maven("https://cache-redirector.jetbrains.com/maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-ide-plugin-dependencies")
 
         maven("https://cache-redirector.jetbrains.com/intellij-repository/releases")
         maven("https://cache-redirector.jetbrains.com/intellij-third-party-dependencies")
 
+        mavenCentral()
+        google()
+
```

Comment:
Unrelated change: put the JB cached repositories first, otherwise Gradle will find the dependency in Maven Central / Google and then not search the cached repositories.